File: app/src/main/java/com/platdmit/simplecloudmanager/data/api/rest/RestSize.java
package com.platdmit.simplecloudmanager.data.api.rest;

import com.platdmit.simplecloudmanager.data.api.models.ApiRequestBody;

import retrofit2.Call;
import retrofit2.http.GET;

public interface RestSize {

    @GET("sizes")
    public Call<ApiRequestBody> getSizes();
}

File: app/src/main/java/com/platdmit/simplecloudmanager/data/api/rest/RestSize.kt
package com.platdmit.simplecloudmanager.data.api.rest

import com.platdmit.simplecloudmanager.data.api.models.ApiRequestBody
import retrofit2.Call
import retrofit2.http.GET

interface RestSize {
    @GET("sizes")
    fun getSizes(): Call<ApiRequestBody>
}